src/worker/storage-models.ts

```diff
@@ -2,7 +2,7 @@
  * @file Interfaces used in storage
  */
 
-import { WordList, ProjectMetadata } from "@common/types";
+import type { WordList, ProjectMetadata } from "@common/types";
```

Comment:
FYI: `import type` is _not_ required in regular `.ts` files. It's just a hack that's required for `.svelte` files :/

However... it is nicely self-documenting... so maybe keep this line as it is! 😄